specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6402,6 +6446,34 @@
         }
       }
     },
+    "InvoiceRebillDetails": {
+      "description": "The properties of a payment.",
+      "properties": {
+        "rebillDocumentType": {
+          "description": "The document type of the invoice in a Enterprise Agreement rebilled invoice relationship.",
```

Comment:
The type of the document generated when an Enterprise Agreement invoice was rebilled,